src/containers/my-quizzes/QuizzesContainer.tsx

```diff
@@ -58,14 +56,14 @@ const QuizzesContainer = () => {
     })
   }, [itemsPerPage, sort, searchTitle, page, selectedItems])
 
-  const deleteQuiz = useCallback(
-    async (id?: string): Promise<AxiosResponse<unknown>> => {
-      const response = await ResourceService.deleteQuiz(id ?? '')
-      await queryClient.invalidateQueries({ queryKey: ['quizzes'] })
-      return response
+  const { mutate: handleDeleteQuiz } = useMutation({
+    mutationFn: ResourceService.deleteQuiz,
+    onError: handleErrorAlert,
+    onSuccess: () => {
+      handleSuccessAlert(`myResourcesPage.quizzes.successDeletion`)
```

Comment:
We don't need these quote here, just use single quote

```suggestion
      handleSuccessAlert('myResourcesPage.quizzes.successDeletion')
```